test-utils/state-viewer/src/main.rs

```diff
@@ -230,26 +236,12 @@ fn main() {
             }
         }
         ("dump_state", Some(_args)) => {
-            let (runtime, state_roots, height) = load_trie(store.clone(), home_dir, &near_config);
+            let (runtime, state_roots, header) = load_trie(store.clone(), home_dir, &near_config);
```

Comment:
separately from this (and doesn't matter right now), but for multiple shards - this can be done only when all shards are tracked (or we need to figure out how to piece from multiple files of genesis per shard). 

Let's add a test though when not all shards are present (and check with human error if that's not true).